### PR TITLE
Staking page new design

### LINKED ISSUE #44: Fix: Community site - Staking page - Add latest design changes
A couple of boxes are missing from the community site that are included in the latest design: 

- TARA rewards -> Lifetime staking rewards
- Validators delegated
- Un-Delegated stake
- Last week’ annualized staking yield

We should change the way we let users know that they SHOULD delegate: right now it’s “Delegate” box with staked un-delegated TARA, that says “you must delegate...” and a button “Delegate to validators” that redirects users to Delegation page.

There’s a couple of reasons why we should update this. Most importantly - this box might confuse people cause we use this kind of box only for action with tokens and NOT for redirecting users to somewhere else.
More appropriate for this purposes would be adding a notice to the top.

https://community.taraxa.io/staking

image

### PR DESCRIPTION
**Pull request by farebord**
Fixes #44

### CODE REVIEW

**Comment by rattrap** [comment]
image

**Comment by farebord** [comment]
Fixed. Needed to parse the ownDelegation stake before substracting

File: services/community-site/src/pages/Staking/Staking.tsx [hunk 1]
```diff
@@ -465,56 +488,80 @@ function Stake({
...
+          title={formatEth(roundEth(weiToEth(unDelegatedStake)))}
           tooltip={
             <Tooltip
               className="staking-icon-tooltip"
-              title="Effective annualized yield, this could be different than the stated expected yields due to special community events. "
+              title="Total number of TARA staked currently that weren't delegated to any validators."
               Icon={InfoIcon}
             />
           }
+          description="Undelegated stake"
```
Comment (VargaElod23) [inline services/community-site/src/pages/Staking/Staking.tsx:564]:
```suggestion
          description="Un-Delegated stake"
``` 
typo

**Review by VargaElod23 — APPROVED** [review]
Small typo, besides that matches what was expected 👍🏼

**Pull request merged**